app/helpers/search_helper.rb

```diff
@@ -14,4 +14,56 @@ def view_online(result)
 
     link_to 'View online', result['sourceLink'], class: 'button button-primary green'
   end
+
+  def view_record(record_id)
+    link_to 'View full record', record_path(id: record_id), class: 'button button-primary green'
+  end
+
+  def number_in_results(result)
```

Comment:
This appears to be adding numbers to an unordered list. I wonder if we could instead use ordered list and update the `start` attribute calculated instead of this type of calculation.

If `ol` doesn't make sense, I wonder if we can move this into the main results loop instead of looping all the results inside the already looping results loop.